includes/functions.php

```diff
@@ -7,6 +7,12 @@ function wpbdp_get_version() {
     return WPBDP_VERSION;
 }
 
+/**
+ * Check pages that have the shortcode and are public.
+ * This is used in the main query to ensure that the views for the main pages are loaded
+ *
+ * @since x.x Allow shortcodes to be used in private pages.
```

Comment:
I'm not sure if our package code will be able to replace the version number if there is extra text in the line. Can you move the extra text up?